include/ibm/locks.hpp

```diff
@@ -124,8 +109,6 @@ class Lock
 
     Lock()
     {
-        // Remove the persistent file
-        std::filesystem::remove(fileName);
         lockTable.clear();
```

Comment:
this is not required as well , because we are removing the persistent file , the lockable will be already gone when we restart bmcweb or reboot bmc.

Reply:
I don't see  lockTable.clear(); in upstream.